lib/msprime.c

```diff
@@ -449,6 +450,7 @@ msp_set_simulation_model_dirac(msp_t *self, double psi)
         goto out;
     }
     self->model.params.dirac_coalescent.psi = psi;
+    self->model.params.dirac_coalescent.const_c = 1.0; // hard coded const_c value
```

Comment:
I am not sure about this constant c in equation (2), was meaning to come and discuss this with you